src/babel-plugin.js

```diff
@@ -11,14 +10,6 @@ const { transformTemplateTag } = require('./template-tag-transform');
  * Its goal is to convert code like this:
  *
  * ```js
- * import { hbs } from 'ember-template-imports';
- *
- * const A = hbs(`A`, {...});
- * const B = [__GLIMMER_TEMPLATE(`B`, {...})];
```

Comment:
This line is still relevant, no?

Reply:
Yeah, but more importantly, the input in the comment doesn't match the output in the comment.

Will fix